features/deduper.js

```diff
@@ -0,0 +1,94 @@
+const deduper = {
+  monitoredChannels: [
+    "102860784329052160", // #general
+    "103696749012467712", // #need-help-0
+    "565213527673929729" // #need-help-1
+  ],
+
+  dupeExpireTime: 2 * 60 * 1000, // 2 minutes to reject the duplicate as cross-post (ms)
+  cleanerInterval: 10 * 60 * 1000, // Frees up the memory from 10 to 10 minutes (ms)
+
+  /**
+   * Structure example:
+   * {
+   *   "userIdOne": {
+   *     "channelIdOne": message1,
+   *     "channelIdTwo": message2,
+   *   },
+   *   "userIdTwo": {
+   *     "channelIdOne": message3,
+   *     "channelIdTwo": message4,
+   *   },
+   * }
+   */
+  lastMessagesOfMember: {},
+
+  deleteMsgAndWarnUser: (msg, lastMsg) => {
+    lastMsg.delete();
+    // prettier-ignore
+    msg.author
+        .send(`Hello there! It looks like you just posted a message to the #${msg.channel.name} channel on our server.
+			
```

Comment:
Is this whitespace intentional?

Reply:
I would imagine it's there to add some spacing between each section of text.

Reply:
Yeah, similar to the message of `#jobs` channel

Reply:
You specifically need three tabs? It can't be just a newline?

Reply:
Oh I didn't even notice it lol
Thanks for pointing out, I removed it